setup.cfg

```diff
@@ -38,6 +38,9 @@ console_scripts =
 pytest11 =
     pytest11.testinfra = testinfra.plugin
 
+[flake8]
+ignore = E501
+
```

Comment:
I have a feeling there are going to a be a lot of failed pylint tests after undoing this and attempting to fix those lines will make the code really ugly since 80 chars isn't really valid anymore on normal terminals.  Just my 3$.